qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -462,17 +462,6 @@ def process_dfe(self, ids: List[str]) -> float:
         """
         return self._client.process_dfe(ids)
 
-    def target_info(self, target: str) -> Dict[str, Any]:
```

Comment:
This is deleted cause it's not how qiskit users are supposed to be getting target info. They should be using `backend.configuration`